### PR TITLE
Absolute threshold tolerance in degrees

### PR DESCRIPTION
**Pull request by vext01**
Use an absolute threshold tolerance in degrees instead of a percentage, as taking a percentage of degrees centigrade makes little sense, as the scale doesn't start from zero. E.g. 15dC is not half as hot as 30dC.

Will need a squash. OK?

### CODE REVIEW

File: krun/platform.py [hunk 1]
```diff
@@ -477,12 +478,15 @@ def _get_num_cpus(self):
         return int(run_shell_cmd(cmd)[0])
 
     def _read_zone(self, zone):
+
```
Comment (ltratt) [inline krun/platform.py]: Random blank line?

**Comment by ltratt** [comment]
Only one minor comment.

**Comment by vext01** [comment]
Yup.

**Comment by vext01** [comment]
OK to squash?

**Comment by ltratt** [comment]
Squash away!

**Comment by vext01** [comment]
Splat.

**Pull request merged**